src/main/java/vazkii/patchouli/common/multiblock/SerializedMultiblock.java

```diff
@@ -7,11 +7,14 @@
 import net.minecraft.block.properties.IProperty;
 import net.minecraft.block.state.IBlockState;
 import net.minecraft.util.ResourceLocation;
+import vazkii.patchouli.api.VariableHolder;
 
 public class SerializedMultiblock {
 
-	String[][] pattern = new String[0][0];
-	Map<String, String> mapping = new HashMap<>();
+	@VariableHolder
+	public String[][] pattern = new String[0][0];
```

Comment:
You can't have @VariableHolder in String[][], it doesn't know how to parse those yet.